.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,5 @@
-**JIRA**: https://jira.appcelerator.org/browse/TIMOB-[TICKET-NUMBER]
+**JIRA:** https://jira.appcelerator.org/browse/[TICKET]
 
-**Additional description (optional)**: 
+Provide a clear PR title prefixed with `[TICKET]`
```

Comment:
Ok. But let's wire up a bet: I swear that we will receive a PR the next week that begins with literally `[TICKET]` :-)